configurations/kms-ear.yaml

```diff
@@ -0,0 +1,11 @@
+
+scylla_encryption_options: "{ 'cipher_algorithm' : 'AES/ECB/PKCS5Padding', 'secret_key_strength' : 128, 'key_provider': 'KmsKeyProviderFactory', 'kms_host': 'kms_test'}"
+
+# config kms_hosts for AWS KMS service account
+append_scylla_yaml: |
+  kms_hosts:
+      kms_test:
+          master_key: 'alias/kms_encryption_test'
```

Comment:
Where do we get it from?

Reply:
It's a key we manually created.

I guess at a point we should add code that create/delete the key for each test, once we want to do more things with it
like force a roll of that key, I'm guess we don't want to affect any other currently running test.

so yeah hard code key, is just a start to get it working.